schemas/groups/navigation.json

```diff
@@ -476,12 +476,22 @@
     },
     "headingMagnetic": {
       "$ref": "../definitions.json#/definitions/numberValue",
-      "description": "Current magnetic heading of the vessel",
+      "description": "Current magnetic heading of the vessel, equals 'headingRaw adjusted for magneticDeviation'",
+      "units": "rad"
+    },
+    "magneticDeviation": {
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "description": "Magnetic deviation of the compass at the current headingRaw",
+      "units": "rad"
+    },
+    "headingRaw": {
+      "$ref": "../definitions.json#/definitions/numberValue",
+      "description": "Current magnetic heading received from the compass.",
```

Comment:
I would write explicitly that this is *Current magnetic heading of the vessel received from the compass prior to any adjustment for deviation or declination* to tell make the relation to the other headings clear.

To me `headingRaw` is too ambiguous in this context, I think `headingCompass` would make much more sense here. 

To me this is about capturing the traditional three heading values & their companion calibration/adjustment values in the schema. If we need a generic raw/calibrated distinction mechanism I think we need something better than appending *raw* all over the place. query/subscription parameter for example. But I feel that is another issue, as I said.

Reply:
I prefer `headingCompass` as well, and I agree with @timmathews  reservations about `headingRaw` as a solution.